utils/ansible-freeipa.spec.in

```diff
@@ -35,6 +35,7 @@ Features
 - Modules for automount key management
 - Modules for automount location management
 - Modules for automount map management
+- Modules for certificate management
```

Comment:
This should be a separate commit and at also PR.

Reply:
fixed.